src/Components/SmartComponents/CVEs/VulnerabilitiesCves.js

```diff
@@ -51,7 +55,9 @@ class VulnerabilitiesCves extends Component {
 
     componentDidMount() {
         StatusDropdown.setCallback(this.sendRequest);
-        this.apply({ show_all: 'true', sort: '-public_date' });
+        const defaultParams = { show_all: 'true', sort: '-public_date' };
```

Comment:
@jiridostal, passing the boolean as `string` (`show_all: 'true'`)  was intended?